Symfony/CS/Fixer.php

```diff
@@ -312,31 +312,6 @@ public static function getLevelAsString(FixerInterface $fixer)
         return 'symfony';
     }
 
-    protected function stringDiff($old, $new)
```

Comment:
tech. speaking this is a BC break, however it is not a useful service point anyway and the functionality of the method itself has been changed as well. @keradus let me what you here.